### ISSUE TITLE
why lightgbm lambdarank example input file doesn't have query information?

### ISSUE THREAD

**Issue by fword** [Microsoft/LightGBM#433]
Plz delete this

**Comment by fword** [comment]
As we kown lightgbm lambdarank's loss function minimize the wrong pairwise order in the same query.While i find lightgbm lambdarank example input format doesn't have the query information,and this is like point-wise
